lib/tcramer.rb

```diff
@@ -11,6 +12,20 @@ module Tcramer
     "What's your opinion of walruses?",
     'How soon before this is ready for production?',
     'Okay, but if you did have a spec, how long would it take?',
-    '(entering a room) dun dun DUN'
+    '(entering a room) dun dun DUN',
+    %{             __ ___'
```

Comment:
Have you considered splitting this out to an `Ascii` module or class? I'm getting concerned about mixing our ascii art and basic text. How can we ensure that Ascii artist's licensing and rights are properly respected.

Reply:
@mejackreed It's a good idea. What would you like to see in the `Ascii` module or class? Can we flesh this out in a new issue? I'm reluctant to jump into a new abstraction in this PR without considering overall systems design. Is that OK?